pkg/storage/export/export/export_test.go

```diff
@@ -1288,6 +1289,11 @@ var _ = Describe("Export controller", func() {
 					Source: &cdiv1.DataVolumeSource{
 						Blank: &cdiv1.DataVolumeBlankImage{},
 					},
+					SourceRef: &cdiv1.DataVolumeSourceRef{
+						Kind:      "",
+						Name:      "test",
+						Namespace: ptr.To("default"),
```

Comment:
Can you please use the kubevirt pointer?

Reply:
Sure!